content/push/index.textile

```diff
@@ -0,0 +1,81 @@
+---
+title: Push notifications
+meta_description: "Ably delivers push notifications to user devices."
+meta_keywords: "Push, push notifications, Apple push notification service, Google Firebase cloud messaging service, APNs, FMC"
+redirect_from:
+  - /general/push
+  - /rest/push
+  - /rest/versions/v1.1/push
+  - /realtime/push
+  - /realtime/versions/v1.1/push
+  - /general/versions/v1.1/push
+  - /general/versions/v1.0/push
+  - /general/push-notifications
+  - /general/versions/v0.8/push-notifications
+  - /general/smart-notifications
+---
+
+Push notifications notify user devices even when an application is not actively running or open. They deliver information, such as app updates, social media alerts, or promotional offers, directly to the user's screen. Ably sends push notifications to devices using Google's Firebase Cloud Messaging service ("FCM":https://firebase.google.com/docs/cloud-messaging) and Apple's Push Notification Service ("APNs":https://developer.apple.com/notifications/). Push notifications don't require a device to stay connected to Ably. Instead, a device's operating system  maintains its own battery-effficient transport to receive notifications.
+
+You can publish push notifications to user devices "directly":publish/#direct-publishing or "via channels":#via-channels.
+
+*Publishing directly* sends push notifications to specific devices identified by unique identifiers, such as a @deviceId@ or a @clientId@. This approach is akin to sending a personal message or alerts directly to an individual user's device, bypassing the need for channel subscriptions. It excels in targeted and personalized communications, such as alerts specific to a user's actions, account notifications, or customized updates.
+
+*Publishing via channels* uses a Pub/Sub model. Messages are sent to channels to which multiple devices can subscribe. When a message is published to a channel, all devices subscribed to that channel receive the notification. This approach is particularly powerful for simultaneously publishing messages to multiple users.
```

Comment:
Adding a link out to different Pub/Sub documentation here would be cool - just in case this is a user's first documentation they read and they don't know what channels are etc..